### PR TITLE
feat: adds logo, icons for flwrs & repo

### PR DESCRIPTION
**Pull request by joe733**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
+
+  <title>Hacking NASA</title>
+</head>
+
+<body>
+  <p align="center">
+    <img src="https://upload.wikimedia.org/wikipedia/commons/e/e5/NASA_logo.svg" width="20%" />
+  </p>
+  <h1>Hackers on the way to hack NASA</h1>
+  <h5>With HTML</h5>
```
Comment (AbhayVAshokan) [inline src/index.html:17]: I think this can be avoided. It's self explanatory I guess. Remove the css part as well.

**Review by AbhayVAshokan — CHANGES_REQUESTED** [review]
Also, could you run `yarn format` before commit. It automatically formats all code.

File: src/index.html (hunk 1 above)
Comment (AbhayVAshokan) [reply src/index.html:17]: Everything else looks great.

Comment (joe733) [reply src/index.html:17]: Which one exactly?
- Logo?
- "with HTML"?
- Both?

Comment (AbhayVAshokan) [reply src/index.html:17]: I meant the "with html" part. On second thoughts, it looks good. Resolving the review.

**Comment by AbhayVAshokan** [comment]
Merging whenever you're ready.

**Comment by joe733** [comment]
You're a go!

**Pull request merged**